rikaicontent.js

```diff
@@ -750,6 +750,14 @@ var rcxContent = {
 		sel.removeAllRanges();
 		sel.addRange(range);
 		tdata.selText = sel.toString();
+
+		if(window.rikaichan.config.ttsEnabled == 'true') {
+			var text = sel.toString();
+			if(typeof text === 'string' && text.length > 0) {

```

Comment:
is it possible for text to not be string or is it just that text may be undefined?

Reply:
I couldn't find a time this failed and https://developer.mozilla.org/en-US/docs/Web/API/Selection/toString says that it must be a string so let's remove.